### ISSUE TITLE
how to ignore self tags in SFC build

### ISSUE THREAD

**Issue by MrGaoGang** [vuejs/vue-loader#1595]
### What problem does this feature solve?
hello, i have i structure in my SFC, such as:

```html
<template>
  <button :style="styles" @click="handleClick">
  </button>
</template>

<script>

</script>

<docs>
   This button is amazing, use it responsibly.
</docs>
```

### What does the proposed API look like?
i have a self define `docs` tags, but i find when i use `vue cli` to build project; the `docs` tag also build to js files, so how should i ignore the tag when i build project!

thank!

**Comment by sodatea** [comment]
Can't reproduce. Please provide a runnable reproduction.
See documentation at https://vue-loader.vuejs.org/guide/custom-blocks.html#example
I believe this block should have been ignored silently.

**Comment by elevatebart** [comment]
@sodatea, I agree that this is what [the documentation](https://vue-loader.vuejs.org/guide/custom-blocks.html#custom-blocks) says. But [here is a quick repro](https://github.com/bodograumann/vue-storybook-docs-page-test) (with storybook). 
You might as well look for the webpack config. It is mostly [here](https://github.com/storybookjs/storybook/blob/master/app/vue/src/server/framework-preset-vue.ts).

**Comment by elevatebart** [comment]
I believe it might have to do with having multiple loaders matching vue files.

**Comment by pksunkara** [comment]
I think the quick reproduction can be done by just having vue-loader and the vue-addon-info loader by @pocka
